uv_copy_and_paste_uv/cpuv_transfer_uv_operation.py

```diff
@@ -87,47 +97,57 @@ def execute(self, context):
 
         uv_layer = bm.loops.layers.uv.verify()  # another approach
 
-        all_sorted_faces = main_parse(self, active_obj, bm, uv_layer)
-        if all_sorted_faces:
-            # check amount of copied/pasted faces
-            if len(all_sorted_faces) != len(props.topology_copied):
-                self.report({'WARNING'}, "Mesh has different amount of faces!!")
-                return {'CANCELLED'}
+        # get selection history
+        all_sel_faces = [e for e in bm.select_history if isinstance(e, bmesh.types.BMFace) and e.select]
+        print(all_sel_faces)
+        if len(all_sel_faces) % 2 != 0:
```

Comment:
I think this condition is weak.
If number of selected faces is even,  this will be problem.